src/main/java/core/basesyntax/service/impl/ParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.LineData;
+import core.basesyntax.service.Parser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParserImpl implements Parser {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private final ValidatorImpl validator;
```

Comment:
```suggestion
    private final Validator validator;
```
варто дотримуватись принципу DI